### DISCUSSION TITLE
calculate dice of different class separately

### DISCUSSION THREAD

**Discussion by YingdiZhang** [Project-MONAI/MONAI#3627] [Q&A] [answered]
Thanks for the excellent work.
I am using MONAI for 3D multi-class segmentation task. When it comes to validation part， I want to calculate the dice metric of every class separately instead of all class together in the tutorial. I wonder how to achive that efficiently？

**Comment by Nic-Ma** [comment, marked as answer]
Hi @yingdiLee ,

Thanks for your interest here.
You can try to set the `reduction` arg of dice metric.

Thanks.
